x-pack/legacy/plugins/canvas/common/lib/autocomplete.ts

```diff
@@ -154,17 +152,46 @@ export function getAutocompleteSuggestions(
 function getFnArgAtPosition(
   ast: ExpressionASTWithMeta,
   position: number
-): { ast: ExpressionASTWithMeta; fnIndex: number; argName?: string; argIndex?: number } {
+): {
+  ast: ExpressionASTWithMeta;
+  fnIndex: number;
+  argName?: string;
+  argIndex?: number;
+  argStart?: number;
+  argEnd?: number;
```

Comment:
nit: extract this as an `interface`... that allows us to document them inline, if necessary.